src/unix/fs.c

```diff
@@ -370,10 +437,10 @@ static ssize_t uv__fs_scandir(uv_fs_t* req) {
   if (dents != NULL) {
     int i;
 
-    /* Memory was allocated using the system allocator, so use free() here. */
+    /* Memory was allocated using the system allocator, so use uv__free() here. */
     for (i = 0; i < n; i++)
-      free(dents[i]);
-    free(dents);
+      uv__free(dents[i]);
+    uv__free(dents);
```

Comment:
hold on, this change is on scandir, I think that was fine, because the library function allocates the memory, so we have to free it with `free` and not `uv__free`.